### DISCUSSION TITLE
React Web Application Failing to Connect to AWS IoT

### DISCUSSION THREAD

**Discussion by ghost** [community/community#46991] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello!

I’m currently working on following this GitHub tutorial: https://github.com/aws-samples/aws-amplify-react-iot-pub-sub-using-cp. 

I’m hoping to use my react web application (which will be hosted on AWS Amplify) to access the AWS IoT Core and be able to publish a message on an MQTT topic. However, I’m facing some connectivity issues as detailed below.  

This is the link to my github react web application: https://github.com/u-shriya/amplify_pubsubdemo 

As shown below, in the in the src folder, aws-iot-core-configuration.js contains a policy line that has a “Connect” feature.
<img width="966" alt="image" src="https://user-images.githubusercontent.com/107970343/218169049-74c41f0e-e63a-4408-a09e-b3482da7b651.png">

Despite this feature being included, we are unable to establish a connection when we run our react web application.
<img width="515" alt="image" src="https://user-images.githubusercontent.com/107970343/218168612-328494df-e9ba-40ec-bc05-e8937500166b.png">

I’m confused about what I may be missing in my code. Any guidance would be greatly appreciated!!

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

Also, you are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
